src/main/scala/beam/agentsim/agents/ridehail/BufferedRideHailRequests.scala

```diff
@@ -16,7 +16,7 @@ class BufferedRideHailRequests(
   private var tick: Double = _
   private var triggerId: Long = _
 
-  def newTimeout(tick: Double, triggerId: Long) = {
+  def newTimeout(tick: Double, triggerId: Long): Unit = {
```

Comment:
Note that these annotations aren't exactly required, and aren't deep so probably not huge - however this adds stress to the compiler. Every specific annotation results in faster and faster compiler speed.